appcelerator/titanium_mobile
# [TIMOB-11554] wait for Organizer window to open

[TIMOB-11554](https://jira.appcelerator.org/browse/TIMOB-11554)

Code changes:
--from----
delay 0.5
---to---
repeat
    try
        if (count of windows) is greater than or equal to 1 then exit repeat
    end try
    delay 0.2
end repeat

Test instructions:
Be sure to reproduce before the fix
Make sure Xcode is not open. Launch Xcode and then with no delay launch "osascript xcode_organizer.scpt" in Terminal
See Organizer - Archives tab open.
